app/routes/index.js

```diff
@@ -1,11 +1,17 @@
 const { admin } = require('../auth/permissions')
+const { clearAllDogs } = require('../session/cdo/dog')
+const { setOwnerDetails, setAddress, setEnforcementDetails } = require('../session/cdo/owner')
 
 module.exports = {
   method: 'GET',
   path: '/',
   options: {
     auth: { scope: [admin] },
     handler: (request, h) => {
+      clearAllDogs(request)
```

Comment:
wondering if pulling this out into a clearCdo function might be useful.  Not essential, just a though. Your call.  You may conclude it doesn't add any benefit right now.